feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/pools/searchPool/SearchPoolViewModel.kt

```diff
@@ -136,12 +136,16 @@ class SearchPoolViewModel(
         searchRaw: String,
         pools: List<PoolRvItem>
     ): PlaceholderModel? {
-        return if (TextUtils.isEmpty(searchRaw)) {
-            PlaceholderModel(resourceManager.getString(R.string.common_search_placeholder_default), R.drawable.ic_placeholder)
-        } else if (pools.isEmpty()) {
-            PlaceholderModel(resourceManager.getString(R.string.search_pool_no_pools_found_placeholder), R.drawable.ic_planet_outline)
-        } else {
-            null
+        return when {
+            TextUtils.isEmpty(searchRaw) -> {
```

Comment:
Why do we use `TextUtils.isEmpty()` instead of `searchRaw.isEmpty()`?

Reply:
It's the same but I am just used to it